<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTapp2Class</class>
 <widget class="QMainWindow" name="QTapp2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTapp2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>800</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>待装配货物:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="newProject"/>
    <addaction name="fromExcel"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>表格管理</string>
    </property>
    <addaction name="add_Goods"/>
    <addaction name="fillAll"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>运行</string>
    </property>
    <addaction name="FirstFit"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="help"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newProject">
   <property name="text">
    <string>新建项目</string>
   </property>
  </action>
  <action name="add_Goods">
   <property name="text">
    <string>添加货物</string>
   </property>
  </action>
  <action name="FirstFit">
   <property name="text">
    <string>FF算法</string>
   </property>
  </action>
  <action name="fillAll">
   <property name="text">
    <string>随机填充表格</string>
   </property>
  </action>
  <action name="fromExcel">
   <property name="text">
    <string>从Excel导入</string>
   </property>
  </action>
  <action name="help">
   <property name="text">
    <string>帮助文档</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QTapp2.qrc"/>
 </resources>
 <connections/>
</ui>
